Command/Message/schema.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$id": "https://smart-data-models.github.io/datamodel.AutonomousMobileRobot/Command/Message/schema.json",
+  "type": "object",
+  "properties": {
+    "type": { "$ref": "https://fiware.github.io/data-models/specs/Robot/schema.json#/definitions/type" },
```

Comment:
These links are to older repos. It should be pointing to https://smart-data-models.github.io/ ...

Reply:
Changed the links to "https://smart-data-models.github.io/"